### DISCUSSION TITLE
Copilot works on Mac VS Code but refuses to work on Windows VS Code

### DISCUSSION THREAD

**Discussion by alexbunardzic** [community/community#7748] [Copilot Conversations] [answered]
I'm a happy Copilot user on the Mac, but when I attempt to install Copilot on the Windows VS Code (1.62.2) I get the following error:

GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

How do I fix that?

**Comment by ryanjsalva** [comment]
I know this might seem like a silly question, but did you authenticate as the same user?

**Comment by davecheney** [comment, marked as answer]
This error indicates that you are either not part of the beta invite list, or there was an error connecting to api.github.com to request a token to use the Copilot service. Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by bgolubovic** [comment]
I am facing the same issue. 

Output:
[INFO] [default] [2021-12-16T15:50:09.128Z] Experimentation service initialized
[INFO] [default] [2021-12-16T15:50:09.133Z] Feature for copilotaa: false
[INFO] [default] [2021-12-16T15:50:09.133Z] Feature for copilotincludeprompt: none
[INFO] [auth] [2021-12-16T15:50:09.622Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2021-12-16T15:50:09.627Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by mediashock** [comment]
How did you get it to work on Mac? I just downloaded Visual Studio for Mac community edition and I cannot install it. I tried downloading the extension as well and its not working. Doesn't even give me any errors when I try to install it.

**Reply by Krzysztof-Cieslak** [reply]
Visual Studio for Mac is not supported by GitHub Copilot.

You can use Copilot on Mac with VSCode (or IntelliJ).

**Reply by mediashock** [reply]
ahhh I downloaded the wrong one.. I wonder if Xcode will ever have support for this. Thanks fro your help, its working now in VSVODE :)
